src/components/tweet-embed/TweetEmbed.tsx

```diff
@@ -46,23 +46,25 @@ const TweetEmbed = ({ options, tweetId }: Props) => {
 `
 
   return (
-    <View style={{ width: '100%', height }}>
+    <>
       {isLoading && (
         <View style={styles.spinnerContainer}>
           <LoadingSpinner color={neutralLight4} />
         </View>
       )}
-      <WebView
-        onMessage={event => {
-          setHeight(parseInt(event.nativeEvent.data, 10))
-          setIsLoading(false)
-        }}
-        scrollEnabled={false}
-        source={{
-          html
-        }}
-      />
-    </View>
+      <View style={{ width: '100%', height }}>
+        <WebView
+          onMessage={event => {
+            setHeight(parseInt(event.nativeEvent.data, 10))
+            setIsLoading(false)
+          }}
+          scrollEnabled={false}
+          source={{
+            html
```

Comment:
Can we move the html code into here so it's not inside a string?

Reply:
Unfortunately react-native-webview only accepts a string for the `html` prop https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#source

We have to do something similar (albeit with js code) in https://github.com/AudiusProject/audius-mobile-client/blob/master/src/components/oauth/OAuth.tsx because the metro bundler doesn't support loading html/js files as a string. Maybe something like this could be possible using handlebars?